cmd/cli/multiregion/failover.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright © 2023 SIL International
+*/
+
+package multiregion
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/silinternational/tfc-ops/v3/lib"
+	"github.com/spf13/cobra"
+)
+
+const (
+	Core                   = "000-core"
+	Cluster                = "010-cluster"
+	ClusterSecondary       = "010-cluster-secondary"
+	Database               = "020-database"
+	DatabaseSecondary      = "020-database-secondary"
+	Ecr                    = "022-ecr"
+	Phpmyadmin             = "030-phpmyadmin"
+	PhpmyadminSecondary    = "030-phpmyadmin-secondary"
+	EmailService           = "031-email-service"
+	EmailServiceSecondary  = "031-email-service-secondary"
+	DbBackup               = "032-db-backup"
+	IdBroker               = "040-id-broker"
+	IdBrokerSecondary      = "040-id-broker-secondary"
+	IdBrokerSearch         = "041-id-broker-search"
+	PwManager              = "050-pw-manager"
+	PwManagerSecondary     = "050-pw-manager-secondary"
+	Simplesamlphp          = "060-simplesamlphp"
+	SimplesamlphpSecondary = "060-simplesamlphp-secondary"
+	IdSync                 = "070-id-sync"
+	IdSyncSecondary        = "070-id-sync-secondary"
+)
+
+const awsFailoverActive = "aws_failover_active"
+
+type Failover struct {
+	testMode bool
+	tfcOrg   string
+
+	workspaces map[string]Workspace
+}
+
+type Workspace struct {
+	lib.Workspace
+	variables []lib.Var
+}
+
+func InitFailoverCmd(parentCmd *cobra.Command) {
+	failoverCmd := &cobra.Command{
+		Use:   "failover",
+		Short: "Failover to secondary region",
+		Long:  `Make Terraform, AWS, and Cloudflare changes for failover to secondary region`,
+		Run: func(cmd *cobra.Command, args []string) {
+			runFailover()
+		},
+	}
+
+	parentCmd.AddCommand(failoverCmd)
+}
+
+func runFailover() {
+	pFlags := getPersistentFlags()
+
+	lib.SetToken(pFlags.token)
+
+	fmt.Println(`Please confirm activation of failover mode. Type "yes" to continue.`)
+	var prompt string
+	_, _ = fmt.Scanln(&prompt)
+	if prompt != "yes" {
+		return
+	}
+
+	f := newFailover(pFlags)
+
+	f.setFailoverActiveVariable("true")
+	f.createRun(Core, "set "+awsFailoverActive+" to true")
```

Comment:
This will be for `Cluster` now, right?

Reply:
That's correct now that I've merged that change on idp-terraform.